packages/base-controller/src/BaseControllerV2.test.ts

```diff
@@ -73,7 +73,12 @@ class CountController extends BaseController<
       state: Draft<CountControllerState>,
     ) => void | CountControllerState,
   ) {
-    super.update(callback);
+    const res = super.update(callback);
+    return res;
+  }
+
+  applyPatches(patches: Patch[]) {
```

Comment:
Nit: If this method is already available in BaseController we don't need to override it.

Reply:
Yeah, I was trying to use it w/o override, but TS was still complaining 🤔